security/security-design/shared-assets/oci-security-health-check-standard/files/oci-security-health-check-standard/requirements.txt

```diff
@@ -1,4 +1,4 @@
-urllib3==1.26.17
+urllib3==1.26.18
```

Comment:
This is a workaround for Cloud Shell.
Python urllib3 is compiled with Linux openssl. For Cloud Shell urllib3 must be of version 1. By default Cloud Shell comes with urllib3 which is compiled with an openssl version not available for Cloud Shell.